src/scripts/clipperUI/components/previewViewer/editorPreviewComponentBase.tsx

```diff
@@ -75,7 +75,8 @@ export abstract class EditorPreviewComponentBase<TState extends EditorPreviewSta
 			changeFontFamily={this.changeFontFamily.bind(this)}
 			changeFontSize={this.changeFontSize.bind(this)}
 			serif={this.props.clipperState.previewGlobalInfo.serif}
-			textHighlighterEnabled={this.props.clipperState.previewGlobalInfo.highlighterEnabled} />;
+			textHighlighterEnabled={this.props.clipperState.previewGlobalInfo.highlighterEnabled}
+			currentMode={this.props.clipperState.currentMode} />;
```

Comment:
where is currentMode being set? Are there two different class for these modes, since in mock test I see only augmentation.

Reply:
These stated are set when any of the options is selected in ModeButtonSelectorClass class . Yes in mock test currently Augmentation mode is used for testing (anyways augmentation and select options are using same base class for header)